central/alert/service/service_impl.go

```diff
@@ -172,8 +179,53 @@ func (s *serviceImpl) GetAlertsCounts(ctx context.Context, request *v1.GetAlerts
 	if request == nil {
 		request = &v1.GetAlertsCountsRequest{}
 	}
+
 	request.Request = ensureAllAlertsAreFetched(request.GetRequest())
-	alerts, err := s.dataStore.ListAlerts(ctx, request.GetRequest())
+	var requestQ *v1.Query
+	if request.GetRequest().GetQuery() == "" {
+		requestQ = search.NewQueryBuilder().AddStringsHighlighted(search.Cluster, search.WildcardString).ProtoQuery()
+	} else {
```

Comment:
Drop the if block altogether. When there is no query only cluster values are pulled out without severity and category.